internal/infrastructure/db/migrations/20220922010142_integration_features.sql

```diff
@@ -0,0 +1,34 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+CREATE TABLE IF NOT EXISTS device_definitions_api.integration_features
+(
+    feature_key         varchar(50) PRIMARY KEY not null,
+    elastic_property    varchar(50) not null,
+    display_name        varchar(50) not null,
+    css_icon            varchar(100),
+    created_at          timestamptz not null default current_timestamp,
+    updated_at          timestamptz not null default current_timestamp
+);
+CREATE UNIQUE INDEX elastic_property_idx ON device_definitions_api.integration_features (elastic_property);
+CREATE UNIQUE INDEX css_icon_idx ON device_definitions_api.integration_features (css_icon);
```

Comment:
I'm not sure I would enforce unique on the css icon, i could see them re-using an icon idk.